### PR TITLE
Fix/ui flows workflows statusview

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Shows flow and workflow diagrams without any runs
- Shows 'No Run' message in the dropdown in the runs view
- Fixes the template being re-used between /runs & /runs/runid states as both are exactly same except ```$state.params.runid``` availability.
- Fixes minor naming conventions

### CODE REVIEW

File: cdap-ui/app/features/flows/controllers/tabs/runs/runs-detail-ctrl.js [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+      if (match.length) {
+        $scope.runs.selected = match[0];
+      }
+    }
+
+    if ($scope.runs.length) {
+      dataSrc.poll({
+        _cdapNsPath: basePath + '/runs/' + $scope.runs.selected.runid
+      }, function(res) {
+        var startMs = res.start * 1000;
```
Comment (anwar6953) [inline cdap-ui/app/features/flows/controllers/tabs/runs/runs-detail-ctrl.js]: indent is off

Comment (ajainarayanan) [inline cdap-ui/app/features/flows/controllers/tabs/runs/runs-detail-ctrl.js]: Fixed.

**Comment by anwar6953** [comment]
LGTM

File: cdap-ui/app/features/workflows/controllers/tabs/runs/runs-detail-ctrl.js [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+    }
+    if ($scope.runs.length) {
+      dataSrc.poll({
+        _cdapNsPath: basePath + '/runs/' + $scope.runs.selected.runid
+      }, function(res) {
+          startMs = res.start * 1000;
+          $scope.startTime = new Date(startMs);
+          $scope.status = res.status;
+          $scope.duration = (res.end ? (res.end * 1000) - startMs : 0);
+          console.info($scope.startTime, $scope.status, $scope.duration);
```
Comment (anwar6953) [inline cdap-ui/app/features/workflows/controllers/tabs/runs/runs-detail-ctrl.js]: oh except this?

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**